### ISSUE TITLE
/journeys earlierRef and laterRef can not be serialized

### ISSUE THREAD

**Issue by ghost** [public-transport/hafas-rest-api#3]
`earlierRef` and `laterRef` are set as attributes of an array by hafas-client. It is impossible to serialize this format into JSON, as JSON arrays can not carry extra attributes.

Possible solutions:
- Change the API in hafas-client to something like this:
```
{
    journeys: [
        [...]
    ],
    earlierRef: "[...]",
    laterRef: "[...]"
}
```

- Or keep the API in hafas-client, but change it to the above in the rest wrapper

**Comment by ghost** [comment]
I implemented the second one here, but of course this would break the rest API:
https://github.com/petabyteboy/hafas-rest-api/commit/2b83bc2002a84c921a84943fc75ea6e0d0fba70f

**Comment by derhuerst** [comment]
[I've put this on my `hafas-client@4` to-do list.](
https://github.com/public-transport/hafas-client/issues/86#issuecomment-445413955)

**Comment by derhuerst** [comment]
c3c7633 has adapted `hafas-rest-api` to [`hafas-client@4`, which returns an object just like you proposed](https://github.com/public-transport/hafas-client/blob/4/docs/journeys.md#response). Published as `hafas-rest-api@2.0.0`.
